src/ice_transport/mod.rs

```diff
@@ -322,6 +326,25 @@ impl RTCIceTransport {
         }
     }
 
+    pub(crate) async fn collect_stats(
+        &self,
+        collector: &Arc<Mutex<StatsCollector>>,
+        worker: Worker,
+    ) {
+        let mut internal = self.internal.lock().await;
+        if let Some(_conn) = internal.conn.take() {
+            let collector = collector.clone();
+            let stats = ICETransportStats::new("ice_transport".to_owned());
+            // TODO: get bytes out of Conn.
+            // bytes_received: conn.bytes_received,
+            // bytes_sent: conn.bytes_sent,
+
```

Comment:
I don't see any mechanism for getting `bytes_received` or `bytes_sent` from the conn.